packages/web-components/src/components/va-file-input/va-file-input.tsx

```diff
@@ -114,6 +127,29 @@ export class VaFileInput {
     this.el.shadowRoot.getElementById('fileInputField').click();
   };
 
+  private removeFile = () => {
+    this.file = null;
+    this.uploadStatus = 'idle';
+  }
+
+  private changeFile = () => {
+    if (this.fileInputRef) {
+      this.fileInputRef.click();
+    }
+  }
+
+  private handleDrop = (event: DragEvent) => {
+    event.preventDefault();
+    event.stopPropagation();
+
+    const files = event.dataTransfer.files;
+    if (files.length > 0) {
+      this.file = files[0];
+      this.vaChange.emit({ files: [this.file] });
+      this.uploadStatus = 'success';
+    }
+  }
+
```

Comment:
I'm not sure if our web component has always been this way so forgive me if I'm missing something, but with file upload, I'd expect to see files stored in a `FileList` on the input. For some reason it's returning an empty list after upload: 

![image](https://github.com/department-of-veterans-affairs/component-library/assets/15097156/72faade3-ae2b-4ae3-8fab-760ef65e86d6)